test/toDoController.spec.js

```diff
@@ -0,0 +1,29 @@
+describe('ToDoController', function() {
+  beforeEach(module('ToDo'));
+
+  var ctrl;
+
+  beforeEach(inject(function($controller) {
+    ctrl = $controller('ToDoController');
+  }));
+
+  it('initialises with an empty to do list', function() {
+    expect(ctrl.todos).toBeUndefined();
+  });
+
+
+  describe('see current to do list ', function() {
+
+    var todos = [
+      {
+        'title': 'Finish weekend challenge', 'done':false
+      }
+    ];
+
+    it('displays a list of todos', function() {
+      ctrl.addToDo = "Finish weekend challenge";
+      expect(ctrl.displayList).toEqual(todos);
+    });
+  });
+
```

Comment:
unit tests look pretty good, aside from the false positive.

would have been nice to also see some protractor tests